src/StreamingBodyParser/ContentLengthBufferedSink.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace React\Http\StreamingBodyParser;
+
+use React\Http\Request;
+use React\Promise\Deferred;
+use React\Promise\PromiseInterface;
+
+class ContentLengthBufferedSink
+{
+    /**
+     * @var Deferred
+     */
+    protected $deferred;
+
+    /**
+     * @var Request
+     */
+    protected $request;
+
+    /**
+     * @var string
+     */
+    protected $buffer = '';
```

Comment:
Please let it `protected`. There's no need to make it private except you want to explicitly forbid to reuse your code.